userspace/libsinsp/container_info.h

```diff
@@ -52,6 +52,14 @@ static inline bool is_docker_compatible(sinsp_container_type t)
 class sinsp_container_lookup
 {
 public:
+	sinsp_container_lookup(const short max_retry = 3, const short max_delay_ms = 500):
+		m_max_retry(3),
```

Comment:
I suppose this should read
```suggestion
		m_max_retry(max_retry),
```

Reply:
indeed, thx!